glass/CMakeLists.txt

```diff
@@ -43,7 +42,6 @@ set_target_properties(libglassnt PROPERTIES DEBUG_POSTFIX "d")
 if(NOT MSVC)
     set_target_properties(libglassnt PROPERTIES OUTPUT_NAME "glassnt")
 endif()
-set_property(TARGET libglassnt PROPERTY POSITION_INDEPENDENT_CODE ON)
```

Comment:
This isn't necessarily redundant; if building with static libraries, this allows linking the resulting static library into a shared library.

Reply:
This is set globally.